### PR TITLE
Foreign key

### PR DESCRIPTION
**Pull request by The-Normal-One1**
### The following lists were made to add foreign keys:

- [x] Created a table named `owners`
- [x] Created a table named `species`
- [x] Modified `animals` table
- [x] Inserted data into the `owners` table
- [x] Inserted data into the `species` table
- [x] Modified my inserted animals so it includes the `species_id` value
- [x] Modified my inserted animals to include owner information `(owner_id)`
- [x] Wrote queries (using JOIN) to answer the following questions

1. What animals belong to Melody Pond?
Melody Pond

2. List of all animals that are pokemon (their type is Pokemon).
listofPokemon

3. List all owners and their animals, remember to include those that don't own any animal.
Owners with all animals

4. How many animals are there per species?
animals per species

5. List all Digimon owned by Jennifer Orwell.
ownedbyJeny

6. List all animals owned by Dean Winchester that haven't tried to escape.
null

7. Who owns the most animals?
Mostowner

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -11,4 +11,70 @@ PRIMARY KEY(id)
...
+
+-- EXTRA INSERT IN OWNERS TABLE
+
+INSERT INTO owners (full_name, age) VALUES
+('Sam Smith', 34),
+('Jennifer Orwell', 19),
+('Bob',45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
```
Comment (codecaiine) [inline schema.sql]: - [x] Please make sure that all thing that populates tables with data should are saved in the data.sql file as required for this project for example query that is beginning by **INSERT** 👍

File: schema.sql [hunk 2]
```diff
@@ -11,4 +11,70 @@ PRIMARY KEY(id)
...
+
+SELECT * FROM owners
+
+-- INSERT DATE FOR SPECIES TABLE
+
+SELECT * FROM species
+
+INSERT INTO species(name) VALUES
```
Comment (codecaiine) [inline schema.sql]: - [x] Please make sure that all thing that populates tables with data should are saved in the data.sql file as required for this project 👍

File: schema.sql [hunk 3]
```diff
@@ -11,4 +11,70 @@ PRIMARY KEY(id)
 );
 
 ALTER TABLE animals
-ADD species TEXT;
\ No newline at end of file
+ADD species TEXT;
+
+--  ADDING OWNERS AND SPECIES TABELS
+
+SELECT * FROM animals
```
Comment (codecaiine) [inline schema.sql]: - [x] Please make sure that info that is a query that helps to analyze data are saved in the queries.sql file as required for this project. for example query that is beginning by **SELECT**  👍

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @The-Normal-One1 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights

- Great Database setup ✔️ 
- Queries work as expected ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @The-Normal-One1, 

To highlight 👍🏽 
✔️ Information is placed in appropriate files
✔️ Screenshots are added
✔️ Gitflow is used

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**